rest-service/manager_rest/storage/resource_models.py

```diff
@@ -279,7 +279,7 @@ def deployment(cls):
         return one_to_many_relationship(cls, Deployment, cls._deployment_fk)
 
     deployment_id = association_proxy('deployment', 'id')
-    blueprint_id = association_proxy('deployment', 'blueprint_id')
+    blueprint_id = db.Column(db.Text, nullable=True)
```

Comment:
i understand that this isn't a fk because we do want to allow deleting the old blueprints?

Reply:
Indeed, like in the case of updating a dep to new blueprint and deleting the old one because we don't need it, but want the log